src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,61 @@
+import { nanoid } from "nanoid";
+import PropTypes from "prop-types";
+import React, { ReactElement, useRef } from "react";
+
+import "./modal.scss";
+
+type Props = {
+  buttons?: ReactElement[];
+  children?: ReactElement;
+  close?: () => void;
+  title?: string;
+};
+
+export const Modal = ({
+  buttons,
+  children,
+  close,
+  title,
+}: Props): ReactElement => {
+  const descriptionId = useRef(nanoid());
+  const titleId = useRef(nanoid());
+  return (
+    <div className="p-modal" onClick={close}>
+      <div
+        className="p-modal__dialog"
+        role="dialog"
+        aria-labelledby={titleId.current}
+        aria-describedby={descriptionId.current}
+        onClick={(evt) => evt.stopPropagation()}
+      >
+        {title && (
```

Comment:
Here `title` should be turned into a boolean (`!!title` or `Boolean(title)`) or should use a ternary otherwise for some falsy values `title` will be rendered (e.g. if `title` was `0` then it will display "0").